tools/db_crashtest.py

```diff
@@ -174,7 +174,7 @@
     "detect_filter_construct_corruption": lambda: random.choice([0, 1]),
     "adaptive_readahead": lambda: random.choice([0, 1]),
     "async_io": lambda: random.choice([0, 1]),
-    "wal_compression": lambda: random.choice(["none", "zstd"]),
+    "wal_compression": lambda: random.choice(["none", "none"]),
```

Comment:
We could simply do `"wal_compression": "none"`, right? Or just comment out this line (I'm assuming the default is no WAL compression)